<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnCevap">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Cevabi Goster</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Yakit Bilgisi</string>
    </property>
    <widget class="QRadioButton" name="rdDizel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Dizel</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdBenzin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Benzin</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="txtCevap">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>104</width>
      <height>87</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBilgiler">
    <property name="title">
     <string>Araba Bilgiler</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action2_seviye"/>
   </widget>
   <addaction name="menuBilgiler"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action2_seviye">
   <property name="text">
    <string>2.seviye</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
